Gemfile

```diff
@@ -19,3 +19,8 @@ group :development, :test do
   gem 'pry'
   gem 'generator_spec'
 end
+
+group :test do
+  gem 'test-unit'
```

Comment:
I'd prefer to get to the bottom of this, rather than brute-forcing a dependency that really shouldn't be necessary. bundle rspec on master works fine for me.

We could add it for now just to get this PR accepted and consider it tech. debt to be addressed later?